### ISSUE TITLE
AnWave says that it is already open

### ISSUE THREAD

**Issue by H3ADROOM** [SimonMacer/AnWave#30]
Hi, I've tried to update my AnWave installation from 4.83.17.45 to the lastest version, but it just says that AnWave is already open so it cannot continue. After that AnWave doesn't want to open until reboot. What should I do?

**Comment by H3ADROOM** [comment]
![image](https://github.com/user-attachments/assets/393bb9ef-bc93-4401-9349-95d12074b3ad)

**Comment by SimonMacer** [comment]
> Hi, I've tried to update my AnWave installation from 4.83.17.45 to the lastest version, but it just says that AnWave is already open so it cannot continue. After that AnWave doesn't want to open until reboot. What should I do?

Open Task Manager and find out AnWave.exe process. You can only have one AnWave window running at a time.

**Comment by H3ADROOM** [comment]
> Open Task Manager and find out AnWave.exe process. You can only have one AnWave window running at a time.

The thing is - it is not there, it happens even when I just booted into Windows and opened AnWave - same issue. I tried to launch it using Sandboxie-Plus and it (although very restricted) launched

**Comment by SimonMacer** [comment]
> > Open Task Manager and find out AnWave.exe process. You can only have one AnWave window running at a time.
> 
> The thing is - it is not there, it happens even when I just booted into Windows and opened AnWave - same issue. I tried to launch it using Sandboxie-Plus and it (although very restricted) launched

Try 5.30 > https://github.com/SimonMacer/AnWave/releases/tag/AnWave-v5.30

Sorry. current. I can't reproduce this problem.

**Comment by H3ADROOM** [comment]
> Try 5.30 > https://github.com/SimonMacer/AnWave/releases/tag/AnWave-v5.30

![image](https://github.com/user-attachments/assets/293c72ea-8099-4eb3-bb2b-90704f61c274)

**Comment by H3ADROOM** [comment]
Guess what? I went into "Find handles or DLLs" section in System Informer and I found what caused it. It was Nahimic Service.

**Comment by SimonMacer** [comment]
> Guess what? I went into "Find handles or DLLs" section in System Informer and I found what caused it. It was Nahimic Service.

So will stopping the Nahimic Service solve this problem? I haven't Nahimic Service. But thanks for letting me know about this issue.

**Comment by H3ADROOM** [comment]
Yes, I've stopped it manually and AnWave started working, it was somehow messing around with AnWave's executable.

**Comment by kamisenpai** [comment]
I have the same issue. I'm on windows 24h2 version. "Does not support running multiple AnWave applications". When I look into task manager there's no other AnWave instance. I tried what @H3ADROOM suggested with system informer, but I have no such service. Idk what to do.

**Comment by H3ADROOM** [comment]
Look for other services

**Comment by SimonMacer** [comment]
> I have the same issue. I'm on windows 24h2 version. "Does not support running multiple AnWave applications". When I look into task manager there's no other AnWave instance. I tried what @H3ADROOM suggested with system informer, but I have no such service. Idk what to do.
Try v5.51
https://github.com/SimonMacer/AnWave/releases/tag/AnWave-v5.51

**Comment by SimonMacer** [comment]
@kamisenpai 
Try v5.51
https://github.com/SimonMacer/AnWave/releases/tag/AnWave-v5.51

**Comment by kamisenpai** [comment]
That's the version I tried. "AnWave-2024-5.51.18.30-Windows-x64.exe" It's the first time trying to install AnWave and it's a somewhat fresh install of windows. Not sure how I would go about finding the service that's causing issues.

**Comment by SimonMacer** [comment]
@kamisenpai 

Hope this solves your problem. Try v5.52 > https://github.com/SimonMacer/AnWave/releases/tag/AnWave-v5.52

- Potential fix for an issue that could cause AnWave launch failures by certain services.

**Comment by kamisenpai** [comment]
> @kamisenpai
> 
> Hope this solves your problem. Try v5.52 > https://github.com/SimonMacer/AnWave/releases/tag/AnWave-v5.52
> 
>     * Potential fix for an issue that could cause AnWave launch failures by certain services.

This worked. Thank you very much! <3
